plugin/pkg/admission/priority/admission.go

```diff
@@ -162,6 +177,11 @@ func (p *priorityPlugin) admitPod(a admission.Attributes) error {
 				return fmt.Errorf("failed to get default priority class: %v", err)
 			}
 		} else {
+			pcName := pod.Spec.PriorityClassName
```

Comment:
this check still needs to be limited to create operations. otherwise we can strand 1.11.0 pod data in a state where it cannot be updated

Reply:
sorry, just traced this back, and this function only gets called in create cases:

```
if operation == admission.Create {
	return p.admitPod(a)
}
```

Reply:
Yes, as I said before and you found as well, this function is called only on create.